### DISCUSSION TITLE
显示图片问题

### DISCUSSION THREAD

**Discussion by hb1912** [gaowei-space/markdown-blog#68] [Q&A]
按一般markdown格式添加的图片，在网站上无法显示，路径有什么要求吗？

**Comment by gaowei-space** [comment]
> 按一般markdown格式添加的图片，在网站上无法显示，路径有什么要求吗？

可以发下你的上下文，我看看

**Reply by hb1912** [reply]
- ![QQ截图20240425194904](20240425194904.png)
主要是图片的位置不知道该怎么写，正常是放在md文件相同的文件夹，本地是没问题的，但是网页上显示找不到图片

**Reply by gaowei-space** [reply]
启动命令中的 fdir 参数支持设置文件路径，来加载文件，包括图片。
示例：
在markdown目录创建 public 文件夹，放入一张图片 1.png，就可以这样在文中加载图片 `![img](/public/1.png)`
